evaluation/examples/linux-cpp/agent/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+g++ example_learner.cpp -o example_learner -lzmq
+
```

Comment:
Isn't it a bit confusing, that both this and the other script that creates Docker image are called "build.sh" ?

what about make.sh or build-agent.sh ? (or build-image.sh for the other one?)